src/lib/events/enums.json

```diff
@@ -402,15 +402,15 @@
                         },
                         "2": {
                             "name": "fallback_posctrl",
-                            "description": "fallback to Position mode"
+                            "description": "Position mode"
```

Comment:
Fallback isn't important context?

Reply:
I don't think so. All the field in this enum are fallbacks in the end.
See my screen recording above to see how it looks for the operator. I think in the context it's enough.

Reply:
And that was an option I had though about once. It adds the key words "failsafe action" before the fallback mode:
![image](https://github.com/PX4/PX4-Autopilot/assets/26798987/ee795f60-cae5-4171-8bf9-16d96ee27b38)
I'm not strongly in favor of any, but think the "failsafe action" keywords are not crucial for context.